.github/ISSUE_TEMPLATE/first_rc_checklist.md

```diff
@@ -17,23 +17,23 @@ labels: task
 * [ ] Merge change log changes.
     - [ ] "PR with changelog entries".
 * [ ] Create X.Y release branch.
-* [ ] Pin llvmlite to `>=0.A.0rc1,<0.A+1.0`.
-* [ ] Pin NumPy if needed
-* [ ] Pin tbb if needed
-* [ ] Annotated tag X.Y.Zrc1 on release branch.
+* [ ] Dependency version pinning on release branch
```

Comment:
```suggestion
* [ ] Commit build farm configuration and tag with release version.
* [ ] Dependency version pinning on release branch
```
Suggest that the above is added as a reminder.

Reply:
This has been added for the final release checklist. I believe this should suffice. Tagging the buildfarm for a first RC isn't that useful as often changes need to be made for the final!